handler/src/test/java/io/netty/handler/ssl/OpenSslEngineTest.java

```diff
@@ -533,6 +534,7 @@ protected void mySetupMutualAuthServerInitSslHandler(SslHandler handler) {
         engine.setVerify(SSL_CVERIFY_IGNORED, 1);
     }
 
+    @Disabled("TLSv1 is not supported by BoringSSL anymore")
```

Comment:
This can be reverted once we pulled in https://github.com/netty/netty-tcnative/pull/902 and released a new version